components/ResourceCard.tsx

```diff
@@ -29,7 +30,9 @@ export const ResourceCard = ({
       justify="space-between"
     >
       <Stack spacing="24px">
-        <Heading fontSize="2xl">{resource.title}</Heading>
+  
+        <Heading fontSize="2xl">{resource.link && (<Link href={resource.link}>{resource.title}</Link>)}</Heading> 
```

Comment:
Two thoughts:
- Is it confusing to have the heading be a link and have the link icon at the bottom of the card?
- Checking the Airtable, it looks like all the resources that don't have links are marked as `Do Not Publish` so should we add [a filter](https://support.airtable.com/docs/formula-field-reference#logical-operators-and-functions) [where we get resources](https://github.com/MutualAidNYC/services-lists/blob/release/pages/index.tsx#L44) to not display them? A think the formula would be something like `NOT({status} = 'Do Not Publish')`.

Reply:
Also, I'm p sure this conditional won't render anything in the heading if there's no link - needs a ternary